packages/matchbox/src/components/Inline/Inline.tsx

```diff
@@ -25,7 +25,30 @@ const InnerWrapper = styled('div')`
   ${alignYChildren}
 `;
 
-function Inline(props) {
+type AlignType = 'center' | 'left' | 'right';
+type AlignYType = 'center' | 'top' | 'bottom';
+
+type InlineProps = {
+  children?: React.ReactNode;
+  /**
+   * Positions children horizontally.
+   * Styled-system responsive arrays work here.
+   */
+  align?: ResponsiveValue<AlignType>;
+  /**
+   * Positions children vertically.
+   * Styled-system responsive arrays work here.
+   */
+  alignY?: ResponsiveValue<AlignYType>;
+  /**
+   * Sets the gutter space between children.
+   * Styled-system responsive arrays work here.
+   */
+  space?: ResponsiveValue<SpaceKeys | string>;
```

Comment:
I feel like `| string` kind of defeats the purpose of specifying keys of `space`, might as well just accept strings